ext/POSIX/POSIX.xs

```diff
@@ -23,6 +23,9 @@ static int not_here(const char *s);
 #include <errno.h>
 #ifdef WIN32
 #include <sys/errno2.h>
+#  ifdef __MINGW32__
+#  include <fenv.h>
+#  endif
```

Comment:
I think it would be better to define `i_fenv` in the config files instead of modifying POSIX. It will have the same effect.